packages/transition-backend/src/api/public/NodesAPIResponse.ts

```diff
@@ -39,12 +33,13 @@ export default class NodesAPIResponse extends APIResponseBase<NodesAPIResponseFo
                     id: feature.properties!.id,
                     code: feature.properties!.code,
                     name: feature.properties!.name,
-                    stops: feature.properties!.data.stops.map((stop) => ({
-                        id: stop.id,
-                        code: stop.code,
-                        name: stop.name,
-                        geography: stop.geography
-                    }))
+                    stops:
+                        feature.properties!.data.stops?.map((stop) => ({
```

Comment:
I feel that some day we should encapsulate those conversion in better object oriented structure. (But I don't know what would be best. Anyway, just thoughts for the future.